src/wwwroot/app/directives/candidate-vacancy-info/hire-date-dialog.template.html

```diff
@@ -14,4 +14,12 @@
         date-error-message="Date should be today or less than"
         valid-method="watch"
     />
+    <textarea
+        ng-if="dialogTransferObject.hireDate"
+        ng-model="dialogTransferObject.comment"
+        placeholder="Comment.."
```

Comment:
it's strange, becouse we use translate filter in views for placeholders and it works.
for example, in cadidate edit view for first name input we have next:
<input type="text" ng-model="candidate.firstName" placeholder="{{::'COMMON.FIRST' | translate}}" name="firstName" validator="required, maxlength=50, letters">